### PR TITLE
VULN-679 - updates axios

### PR DESCRIPTION
**Pull request by tradeshift-larsbojensen**

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-8.9.0
+10
```
Comment (wejendorp) [inline .nvmrc:1]: I think we should move it to 14 or 16, as per https://nodejs.org/en/about/releases/

File: package.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
 		"npm": "5.3.0"
 	},
 	"dependencies": {
-		"axios": "0.21.1",
+		"axios": "0.21.2",
```
Comment (wejendorp) [inline package.json:21]: And similarly, we might as well bump to `0.24` unless something breaks.

File: .nvmrc (hunk 1 above)
Comment (tradeshift-larsbojensen) [reply .nvmrc:1]: Yes, I went very conservatively about this. Probably too much, bc 10 has passed away, and 12 is going to soon.

File: package.json (hunk 2 above)
Comment (tradeshift-larsbojensen) [reply package.json:21]: Let's go crazy!

File: .nvmrc (hunk 1 above)
Comment (tradeshift-larsbojensen) [reply .nvmrc]: Just updated to 16, and axios to 0.24.0, seems to run fine.

**Comment by ehats** [comment]
retest

**Review by esmailpour-hosein — APPROVED** [review]

**Review by wejendorp — APPROVED** [review]

**Pull request merged**